React/tic-tac/src/app/screens/Setting/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, Fragment } from 'react';
+import Header from '@components/Header';
+import actions from '@redux/setting/actions';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import IconSelector from './ components/IconSelector';
+import style from './style.scss';
+
+class Setting extends Component {
+  handleSelect(icon, isPlayerOne) {
+    this.props.setIcon(icon, isPlayerOne);
+  }
+
+  selectPlayerOne = icon => {
+    this.handleSelect(icon, true);
+  };
+
+  selectPlayerTwo = icon => {
+    this.handleSelect(icon, false);
+  };
+
+  render() {
+    return (
+      <Fragment>
+        <Header />
```

Comment:
Be carefull with this. Header must be used once, not in each screen that you add. Keep it in mind for upcoming projects

Reply:
I'll keep it in mind